config/jobs/kubernetes-sigs/descheduler/descheduler-presubmits-master.yaml

```diff
@@ -50,11 +50,11 @@ presubmits:
         - build
         resources:
           limits:
-            cpu: 6
-            memory: 4Gi
+            cpu: 8
+            memory: 8Gi
           requests:
-            cpu: 6
-            memory: 4Gi
+            cpu: 8
+            memory: 8Gi
```

Comment:
@a7i I believe we want to makes these changes to lines 24 to 30 for the job `pull-descheduler-verify-master` to fix golangci-lint failing in CI. You've changed job `pull-descheduler-verify-build-master` which I haven't seen fail recently.

Reply:
oh yes, good call! I even referenced pull-descheduler-verify-master dashboards